### PR TITLE
Removed everthing not related to sign-in/out, functionality is…

### PR DESCRIPTION
**Pull request by philipborg**
… uncertain

### CODE REVIEW

File: src/SignIn/Api/BlenderMaps.cs [hunk 1]
```diff
@@ -10,9 +10,6 @@ public void Register()
             Blender.MapUri("/signin/user", "user");
             Blender.MapUri("/signin/signinuser", "userform");
             Blender.MapUri("/signin/signinuser?{?}", "userform-return");
-            Blender.MapUri("/signin/admin/settings", "settings");
-            Blender.MapUri("/signin/user/authentication/password/{?}", "authentication-password");
-            Blender.MapUri("/signin/user/authentication/settings/{?}", "authentication-settings");
             Blender.MapUri("/signin/partial/user/image", "userimage-default");
```
Comment (Mackiovello) [inline src/SignIn/Api/BlenderMaps.cs]: Is this one needed?

File: src/SignIn/Helpers/Utils.cs [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
 using Simplified.Ring3;
-using Simplified.Ring6;
 using Starcounter;
+using System.Net.Mail;
 
 namespace SignIn
 {
     public class Utils
     {
+        public static string RandomString(int size)
```
Comment (Mackiovello) [inline src/SignIn/Helpers/Utils.cs]: What is this one used for?

Comment (philipborg) [inline src/SignIn/Helpers/Utils.cs]: No idea, I will check. I was equally confused as to why that was needed.

File: src/SignIn/Api/BlenderMaps.cs (hunk 1 above)
Comment (philipborg) [inline src/SignIn/Api/BlenderMaps.cs]: userimage-default? I think it might use it internally like it did with /signin/partial/signout

Comment (philipborg) [inline src/SignIn/Api/BlenderMaps.cs]: Hmm, forgot it was blending not Handle. Either way, if we keep the handle we should keep the blending.

File: src/SignIn/Helpers/Utils.cs (hunk 2 above)
Comment (philipborg) [inline src/SignIn/Helpers/Utils.cs]: Removed it

**Pull request merged**